File: C/ex02.c
// Estoque médio de uma peça

#include <stdio.h>

int main() {

    int quant_min, quant_max;
    float media;
    
    printf("Informe a quantidade mínima de estoque: ");
    scanf("%d", &quant_min);
    
    printf("Informe a quantidade máxima de estoque: ");
    scanf("%d", &quant_max);
    
    media = (quant_min + quant_max) / 2;
    
    printf("O estoque médio é: %.2f", media);

    return 0;
}

File: Python/ex02.py
#Estoque médio de uma peça

quantidade_minima =int(input("Informe a quantidade mínima: "))
quantidade_maxima =int(input("Informe a quantidade máxima: "))

estoque_medio = (quantidade_minima + quantidade_maxima) / 2

print("O estoque médio é {0}".format(estoque_medio))